### ISSUE TITLE
Unable to launch in Windows

### ISSUE THREAD

**Issue by LeonisX** [gurkenlabs/litiengine#406]
The utility does not start when several different versions of the JDK are installed on the system.

The problem is that if version 16 is found, then the utility uses this version and there is a problem with modules exports. 

**Stack Trace**

```java
Wed Jun 16 15:33:48 EEST 2021 AWT-EventQueue-0 threw an exception:
java.lang.IllegalAccessError: class com.github.weisj.darklaf.ui.label.DarkLabelUI (in unnamed module @0x7aec35a) cannot access class sun.swing.SwingUtilities2 (in module java.desktop) because module java.desktop does not export sun.swing to unnamed module @0x7aec35a
	at com.github.weisj.darklaf.ui.label.DarkLabelUI.paint(DarkLabelUI.java:83)
	at java.desktop/javax.swing.plaf.ComponentUI.update(ComponentUI.java:161)
	at java.desktop/javax.swing.JComponent.paintComponent(JComponent.java:797)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1074)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:907)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1083)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:907)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1083)
	at java.desktop/javax.swing.JLayeredPane.paint(JLayeredPane.java:586)

```

**To Reproduce**

1. Install JDK 8 and 16
2. Launch utility
3. See empty black screen

**Your System:**
 - OS: Windows 10
 - LITIENGINE version: 0.5.1-beta
 - Java JDK/JRE version: jdk1.8, jdk16
 - Screen resolution: 1920x1080

**Comment by nightm4re94** [comment]
This is not a bug. Executing a java program will default to the jdk which is defined in your system's environment variables. If that happens to be jdk 16, some deprecated modules are no longer available.

**Comment by LeonisX** [comment]
Then how does the utility look for Java?

I have two environment variables defined now:

JAVA_HOME = C: \Program Files\Java\jdk08
PATH= %JAVA_HOME%\bin;...

Also on the recommendation of [this resource](https://www.happycoders.eu/java/how-to-switch-multiple-java-versions-windows) I deleted the following directories:

C:\ProgramData\Oracle\Java\javapath
C:\Program Files (x86)\Common Files\Oracle\Java\javapath

The java -version command says I have version 8.

Is there any Java searching mechanism that I am not aware of?

**Comment by KeshavThosar** [comment]
There's one **hacky** way to deal with the issue. What you can do is:  download the source code from this [https://github.com/gurkenlabs/litiengine/archive/refs/tags/v0.5.1-beta.zip](url). You could find the rest of the releases [https://github.com/gurkenlabs/litiengine/releases](here). After that build the solution and then from the main directory, go to  **uitiliti** > **build** > **libs** where you will find **utiliti-v0.5.1-beta.jar** (Note that the **build** folder will only be present if you built the project on your system). Next

**Comment by weisJ** [comment]
A simpler version:
- Clone the repository
- Run `gradlew jpackage`
- Use the resulting `exe` located in `utiliti/build/jpackage` (Not sure if this is the correct path right now. Simply search in `utiliti/build` for an exe file and it should be the correct one)

**Comment by Gamebuster19901** [comment]
> A simpler version:
> 
> * Clone the repository
> * Run `gradlew jpackage`
> * Use the resulting `exe` located in `utiliti/build/jpackage` (Not sure if this is the correct path right now. Simply search in `utiliti/build` for an exe file and it should be the correct one)

Please note that these instructions assume that the latest commit on master actually builds a usable exe, which may not always be the case.

**Comment by Gamebuster19901** [comment]
I went ahead and created an unofficial release at based on commit 81a1903d8597318f3db41ad943c504fe0d5b38b6 as a temporary solution.

It is located here:  https://github.com/Gamebuster19901/litiengine/releases/tag/v0.5.2-UNOFFICIAL-GAMEBUSTER

**Comment by steffen-wilke** [comment]
Just for the sake of completeness, here is a similar error from the forum post [I have an error then running SDK](https://forum.litiengine.com/d/153-i-have-an-error-then-running-sdk)

```
java.lang.IllegalAccessError: class com.github.weisj.darklaf.graphics.StringPainter (in unnamed module @0x5c29bfd) cannot access class sun.swing.SwingUtilities2 (in module java.desktop) because module java.desktop does not export sun.swing to unnamed module @0x5c29bfd
at com.github.weisj.darklaf.graphics.StringPainter.drawStringUnderlineCharAt(StringPainter.java:85)
at com.github.weisj.darklaf.ui.button.DarkButtonUI.paintText(DarkButtonUI.java:347)
at com.github.weisj.darklaf.ui.button.DarkButtonUI.paint(DarkButtonUI.java:195)
at java.desktop/javax.swing.plaf.ComponentUI.update(ComponentUI.java:161)
at java.desktop/javax.swing.JComponent.paintComponent(JComponent.java:842)
at java.desktop/javax.swing.JComponent.paint(JComponent.java:1119)
at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:952)
at java.desktop/javax.swing.JComponent.paint(JComponent.java:1128)
at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:952)
at java.desktop/javax.swing.JComponent.paint(JComponent.java:1128)
at java.desktop/javax.swing.JLayeredPane.paint(JLayeredPane.java:586)
at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:952)
at java.desktop/javax.swing.JComponent.paint(JComponent.java:1128)
at java.desktop/javax.swing.JComponent.paintToOffscreen(JComponent.java:5311)
at java.desktop/javax.swing.RepaintManager$PaintManager.paintDoubleBufferedImpl(RepaintManager.java:1657)
at java.desktop/javax.swing.RepaintManager$PaintManager.paintDoubleBuffered(RepaintManager.java:1632)
at java.desktop/javax.swing.RepaintManager$PaintManager.paint(RepaintManager.java:1570)
at java.desktop/javax.swing.RepaintManager.paint(RepaintManager.java:1337)
at java.desktop/javax.swing.JComponent._paintImmediately(JComponent.java:5259)
at java.desktop/javax.swing.JComponent.paintImmediately(JComponent.java:5069)
at java.desktop/javax.swing.RepaintManager$4.run(RepaintManager.java:879)
at java.desktop/javax.swing.RepaintManager$4.run(RepaintManager.java:862)
at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
at java.desktop/javax.swing.RepaintManager.paintDirtyRegions(RepaintManager.java:862)
at java.desktop/javax.swing.RepaintManager.paintDirtyRegions(RepaintManager.java:835)
at java.desktop/javax.swing.RepaintManager.prePaintDirtyRegions(RepaintManager.java:784)
at java.desktop/javax.swing.RepaintManager$ProcessingRunnable.run(RepaintManager.java:1898)
at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:771)
at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:716)
at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:741)
at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by steffen-wilke** [comment]
Resolved with the official new release `v0.5.2-beta`
